api/api/validator.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+
+import json
+from xml.etree import ElementTree as ET
+import re
+
+
+regex_dict = {'numbers': r'^\d+$',
```

Comment:
Please, could you sort it alphabetically?

Reply:
Done!